stubs/mongodb.phpstub

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace MongoDB\Driver;
+
+/**
+ * @template-covariant TKey
+ * @template-covariant TValue
+ *
+ * @template-extends Traversable<TKey, TValue>
+ */
+interface CursorInterface extends Traversable
+{
+    /**
+     * @return array<TValue>
+     */
+    public function toArray();
+}
+
+/**
+ * @template-covariant TValue of array|object
+ *
+ * @template-implements Iterator<int, TValue>
+ * @template-implements CursorInterface<int, TValue>
```

Comment:
The types are based on 
`array|object`: https://www.php.net/manual/en/mongodb-driver-cursor.current.php
`int`: https://www.php.net/manual/en/mongodb-driver-cursor.key.php